WORKSPACE

```diff
@@ -13,3 +13,9 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_depe
 go_rules_dependencies()
 
 go_register_toolchains()
+
+http_archive(
+    name = "io_bazel",
+    sha256 = "6860a226c8123770b122189636fb0c156c6e5c9027b5b245ac3b2315b7b55641",
+    url = "https://github.com/bazelbuild/bazel/releases/download/0.22.0/bazel-0.22.0-dist.zip",
```

Comment:
Interesting... will this work with future versions of bazel?

Reply:
The proto definition is changing (see https://github.com/bazelbuild/bazel/issues/6861), and we'll likely have to migrate at some point. It might even be possible to migrate right now and have things work back to at least Bazel 0.21.0, but I wanted to start somewhere and then iterate.

Also, eventually this is supposed to be all Starlark, which should be much easier to generate.